modules/oe_theme_content_publication/oe_theme_content_publication.libraries.yml

```diff
@@ -0,0 +1,5 @@
+publication_css:
```

Comment:
Let's not call the library `publication_css` just because for now it only handles css. Let's name it: `oe_publication` like the content type.